starknet_devnet/account_util.py

```diff
@@ -120,12 +121,18 @@ async def set_balance(state: StarknetState, address: int, balance: int):
 
     fee_token_address = state.general_config.fee_token_address
 
-    balance_address = pedersen_hash(get_selector_from_name("ERC20_balances"), address)
+    balance_key = pedersen_hash(get_selector_from_name("ERC20_balances"), address)
     balance_uint256 = Uint256.from_felt(balance)
 
     await state.state.set_storage_at(
-        fee_token_address, balance_address, balance_uint256.low
+        storage_domain=StorageDomain.ON_CHAIN,
+        contract_address=fee_token_address,
+        key=balance_key,
+        value=balance_uint256.low,
     )
     await state.state.set_storage_at(
-        fee_token_address, balance_address + 1, balance_uint256.high
+        storage_domain=StorageDomain.ON_CHAIN,
+        contract_address=fee_token_address,
+        key=balance_key + 1,
```

Comment:
I don't understand this `balance_key + 1`. What is that?

Reply:
This was not added in this PR, but explained in a comment: https://github.com/0xSpaceShard/starknet-devnet/pull/512/commits/5094cfd6c732ae6d683361b46e1059520894dd08